### PR TITLE
What's new updates for 0.12.0

### PR DESCRIPTION
**Pull request by SueChaplain**
New section for the What's new page
to provide the highlights for the
0.12.0 release.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Jenkins website stage

**Comment by SueChaplain** [comment]
Jenkins website stage

**Comment by SueChaplain** [comment]
Jenkins website stage

File: oj9_whatsnew.html [hunk 1]
```diff
@@ -78,50 +79,95 @@ <h1>What's new?</h1>
         <span class="intro-text">Covering project news, events, milestones, and of course... new and cool stuff!</span>
       </div>
 
-	<div class="f-section-item" id="openj90110">
-    
+    <div class="f-section-item" id="openj90120">
+    <div class="f-content-container">
+      <h3>Eclipse OpenJ9 version 0.12.0 released</h3>
+      <p><i>23rd January 2019</i></p>
```
Comment (pshipton) [inline oj9_whatsnew.html]: This date will need to change

File: oj9_whatsnew.html [hunk 2]
```diff
@@ -78,50 +79,95 @@ <h1>What's new?</h1>
...
+      <p>Our latest version of Eclipse OpenJ9 is now released, with a number of improvements and new benefits. Here are the highlights:
+      </p>
+	  <p><i class="fa fa-hand-o-right" aria-hidden="true" style="color: #407471;opacity: 0.7; font-size:.9rem"></i> <strong>Support for macOS&reg;</strong>
+      </p>
+      <p>This release adds support for OpenJDK 8 with OpenJ9 on the 64-bit macOS platform.</p>
+      <p><i class="fa fa-hand-o-right" aria-hidden="true" style="color: #407471;opacity: 0.7; font-size:.9rem"></i> <strong>Extended support for pause-less garbage collection</strong>
+      </p>
+      <p>Concurrent scavenge mode is now available on 64-bit Windows platforms. Support for 64-bit Linux x86 platforms is now extended for applications that require large heap (non-compressed references) VMs. </p>
+	  <p><i class="fa fa-hand-o-right" aria-hidden="true" style="color: #407471;opacity: 0.7; font-size:.9rem"></i> <strong>Improved cryptographic performance for the RSA algorithm.</strong>
```
Comment (pshipton) [inline oj9_whatsnew.html]: Need to keep an eye on this. It is delivered for jdk11, but not yet jdk8. Undetermined if the jdk8 change will make it into the release.

File: oj9_whatsnew.html (hunk 1 above)
Comment (SueChaplain) [reply oj9_whatsnew.html]: And the content maybe.

**Comment by SueChaplain** [comment]
Jenkins website stage

**Comment by SueChaplain** [comment]
Jenkins website stage

**Comment by SueChaplain** [comment]
Jenkins website stage

**Pull request merged**